src/modules/UI/scenes/CryptoExchange/CryptoExchangeSceneComponent.js

```diff
@@ -191,7 +192,11 @@ export class CryptoExchangeSceneComponent extends Component<Props, State> {
             />
             <View style={style.shim} />
             <View style={style.actionButtonContainer} >
-              {this.renderButton()}
+            <PrimaryButton
```

Comment:
What's the purpose of bypassing `renderButton()`? Seems like you now will show the user the button even if `showNextButton` is `false`. Also, `showNextButton` is no longer used but it is still in the Flow type `CryptoExchangeSceneComponentStateProps`

Reply:
The `processingFlag` doesn't appear to prevent the `PrimaryButton` from showing or being tapped.

Reply:
@paullinator per my note in Asana, it seemed like we should keep the button up all the time, It is a little confusing to go from , not there, to spinning each time we type something. etc.. I have modified the button to not generate a click action if the processingFlag is true

Reply:
It's confusing as well for a button to be there but with no response. Can you do an Alert that pops up the appropriate reason why the button doesn't work?